src/Microsoft.Health.Dicom.Core.UnitTests/Features/Retrieve/RetrieveRenderedServiceTests.cs

```diff
@@ -314,6 +322,8 @@ public async Task GivenStoredInstancesWithFramesPNG_WhenRetrieveRenderedForFrame
         resultStream.Position = 0;
         AssertStreamsEqual(resultStream, response.ResponseStream);
         Assert.Equal("image/png", response.ContentType);
+        Assert.True(_dicomRequestContextAccessor.RequestContext.IsRenderedRequested);
```

Comment:
Could you also assert that PartCount == 1

Reply:
added